### PR TITLE
Fix typo in the dependency name

### PR DESCRIPTION
**Pull request by batmat**
First noticed by @jennyfive, thanks a lot!

### CODE REVIEW

File: distribution/Makefile [hunk 1]
```diff
@@ -95,7 +95,7 @@ container-check-prereqs: shunit2 ./tests/tests.sh ./tests/offline-tests.sh creat
 	$(MAKE) -C ../services lint
 	$(MAKE) -C ../services dump generate-ingest
 
-container-check: docker-cloud-container-check aws-cloud-container-check java11-docker-cloud-check rollback-check
+container-check: docker-cloud-container-check aws-cloud-container-check java11-docker-cloud-container-check rollback-check
```
Comment (batmat) [inline distribution/Makefile]: See L112 below.

**Review by mandie722 — APPROVED** [review]

**Comment by jennyfive** [comment]
@batman :) ! seems like `make check` runs fine on Mac now (although there are a couple of errors), but this particular one is solved!

**Review by alecharp — APPROVED** [review]
:man_facepalming: This is on me.

**Comment by batmat** [comment]
OK, the last failure was because flapmap-stream got depublished so it failed why running `npm ci`, retriggering.

**Pull request merged**